lib/ex_unit/lib/ex_unit/capture_io.ex

```diff
@@ -124,9 +124,18 @@ defmodule ExUnit.CaptureIO do
     {:ok, capture_io} = StringIO.open(input)
     Process.register(capture_io, device)
 
+    flag = Process.flag(:trap_exit, true)
```

Comment:
We don't want to force a trap exit because this is surprising behaviour to a user :(